controller/controller_cancel_crud_test.go

```diff
@@ -317,7 +318,7 @@ func Test_ProcessUpdate_CRUD(t *testing.T) {
 }
 
 type testCRUDResource struct {
-	*CRUDResource
+	*crud.Resource
```

Comment:
These tests here are not super cool. I just kept them working somehow to get rid of the deprecated code.